examples/sycl/pvc/pvc_gemm_fp8.cpp

```diff
@@ -0,0 +1,571 @@
+          out(v, i, j) = static_cast<cutlass::bfloat16_t/*typename TiledMma::ValTypeB*/>(in.data()[B_selector(v, i, j)]);
+          //  out(v, i, j) = (bfloat16_t)(1.0f);
+         }
+       }
+     }
+     #endif
+   }
+  };
+  
+ int main(int argc, const char** argv)
+ {
+   //
+   // Parse options
+   //
+ 
+   Options options;
+ 
+   options.parse(argc, argv);
+ 
+   if (options.help) {
+     options.print_usage(std::cout) << std::endl;
+     return 0;
+   }
+ 
+   if (options.error) {
+     std::cerr << "Aborting execution." << std::endl;
+     return -1;
+   }
+ 
+   //
+   // Run examples
+   //
+ 
+   // The KernelHardwareInfo struct holds the number of EUs on the GPU with a given device ID. This
+   // information is used by the underlying kernel.
+   cutlass::KernelHardwareInfo hw_info;
+ 
+   // Change device_id to another value if you are running on a machine with multiple GPUs and wish
+   // to use a GPU other than that with device ID 0.
+   hw_info.sm_count = cutlass::KernelHardwareInfo::query_device_multiprocessor_count(hw_info.device_id);
+ 
+   bool passed;
+ 
+   // The code section below describes datatype for input, output matrices and computation between
+   // elements in input matrices.
+   using ElementAccumulator = float;                   // <- data type of accumulator
+   using ElementComputeEpilogue = float;               // <- data type of epilogue operations
+   using ElementInputA = cutlass::float_e4m3_t;        // <- data type of elements in input matrix A
+   using ElementInputB = cutlass::float_e4m3_t;        // <- data type of elements in input matrix B
+   using ElementOutput = float;                        // <- data type of elements in output matrix D
+ 
+   using LayoutC = cutlass::layout::RowMajor;
+   using LayoutD = cutlass::layout::RowMajor;
+ 
+   // Note: XE_2D_U8x32x32_LD_V is incompatible with our bf16 MMA atoms
+   // 2.8tflops  U8x32x32NLD_N 
+   // 1.4tflops  U8x16x32NLD_N
+   // 0.7tflops  U8x 8x32NLD_N
+   #if defined(A_COL) && defined(B_ROW)
+   using LayoutA = cutlass::layout::ColumnMajor;
+   using LayoutB = cutlass::layout::RowMajor;
+   using GmemTiledCopyA = XE_2D_U8x16x32_LD_T;
+   using GmemTiledCopyB = XE_2D_U8x32x32_LD_N;
+   #endif
+   #if defined(A_ROW) &&  defined(B_ROW)
+   using LayoutA = cutlass::layout::RowMajor;
+   using LayoutB = cutlass::layout::RowMajor;
+   using GmemTiledCopyA = XE_2D_U8x32x32_LD_N;
+   using GmemTiledCopyB = XE_2D_U8x32x32_LD_N;
+   #endif
+
+   #if defined(A_COL) & defined(B_COL)
+   using LayoutA = cutlass::layout::ColumnMajor;
+   using LayoutB = cutlass::layout::ColumnMajor;
+   using GmemTiledCopyA = XE_2D_U8x16x32_LD_T;
+   using GmemTiledCopyB = XE_2D_U8x16x32_LD_T;
+   #endif
+   
+   #if defined(A_ROW) && defined(B_COL)
+   using LayoutA = cutlass::layout::RowMajor;
+   using LayoutB = cutlass::layout::ColumnMajor;
+   using GmemTiledCopyA = XE_2D_U8x32x32_LD_N;
+   using GmemTiledCopyB = XE_2D_U8x16x32_LD_T;
```

Comment:
Hi, are you not using VNNI format for `B` anywhere?
Do you shuffle `B` data after loading into registers to change layout to VNNI format?

Thanks